breadcord/core_modules/settings_manager/__init__.py

```diff
@@ -129,7 +130,7 @@ async def autocomplete_value(self, interaction: discord.Interaction, current: st
                 return [app_commands.Choice(name=f"⚠️ Invalid integer '{current}'", value=current)]
 
         elif setting.type == str:
-            if len(current) >= 2 and current[0] == current[-1] and current[0] in '\'"':
+            if current[0] + current[-1] in ('""', "''"):
```

Comment:
I'm realising now that this would probably cause an IndexError when `current` is an empty string.

Reply:
> I'm realising now that this would probably cause an IndexError when `current` is an empty string.

Yes, but would it ever be empty? Considering that a setting can’t be, the function would return before getting here.

Reply:
As discussed on Discord, this should be fine as-is - the code here is awful and needs refactoring anyways, and a bug involving broken autocomplete for empty inputs is the least of our concerns.